minethd.cpp

```diff
@@ -33,7 +33,11 @@
 
 void thd_setaffinity(std::thread::native_handle_type h, uint64_t cpu_id)
 {
+#ifdef __MINGW64__
+	SetThreadAffinityMask(&h, 1ULL << cpu_id);
```

Comment:
@ibroheem sorry to drag it out (if I had the time I would have just ran it myself), HANDLEs on Windows are usually passed by value, as you can see in the code on the other side of the #else. 

My point is that the HANDLE = void* makes it possible for your code to fail silently if you pass an address where a value is expected.